app/controllers/events/view/tickets/orders/list.js

```diff
@@ -35,7 +35,7 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
         cellComponent   : 'ui-table/cell/events/view/tickets/orders/cell-date',
         headerComponent : 'tables/headers/sort',
         width           : 100,
-        dateFormat      : 'MMMM DD, YYYY - HH:mm A',
```

Comment:
Pls also correct this declaration, here I think we need to define dateFormat inside options